h2o-r/build.gradle

```diff
@@ -152,11 +162,54 @@ task cpH2OAppJar(type: Copy, dependsOn: [configurations.compile]) {
     into "${projectDir}/h2o-package/inst/java/"
 }
 
+task cpMainPKG {
+    doLast {
+        file("${buildDir}/h2o-package-main").mkdirs()
+        copy {
+            from "h2o-package"
+            into "${buildDir}/h2o-package-main"
+        }
+
+        editConnectionSource("h2o-package-main", "h2o.init()", "After starting H2O, you can use the Web UI" +
```

Comment:
I still don't like that we have this "code" inside of build file. the amount of back-slashes in the strings makes it error prone.
I would at least extract these strings into a `map` at the beggining of the build file.
Ideally I would move these into .txt files somewhere in the h2o-r directory (for example h2o-r/templates/main/start_command.txt, start_command_doc.txt) and add a README into the h2o-r/templates to document how these are used.

Reply:
Your suggestion is implemented [here](https://github.com/h2oai/h2o-3/pull/5135/commits/09ef16e32bf1de4296f9aa97933137e68769e645).

I try also use source.properties file to define variables and values. Implementation is on HEAD of this branch.

Please tell me what you think